### DISCUSSION TITLE
`workflow_dispatch` by `createWorkflowDispatch` API should return `run_id`

### DISCUSSION THREAD

**Discussion by ShaMan123** [community/community#34283] [Actions] [answered]
Hi there,
I love Github and github actions. Amazing product. Thanks!
I am working with octokit rest API, [dispatching workflows](https://docs.github.com/en/rest/actions/workflows#create-a-workflow-dispatch-event).
IMO the end point should return the run id of the workflow for future reference.
I am aware I can [list workflows](https://docs.github.com/en/rest/actions/workflow-runs#list-workflow-runs-for-a-workflow) and find the one I have triggered but it is cumbersome.

**Comment by ajschmidt8** [comment, marked as answer]
FYI, this discussion seems to be a duplicate of https://github.com/community/community/discussions/9752

**Reply by ShaMan123** [reply]
Thanks, it is
